malasakit-django/pcari/templates/base.html

```diff
@@ -36,6 +37,18 @@
         <h1 id="main-heading">{% block main-heading %}Malasakit{% endblock %}</h1>
       </div>
     </header>
+    <page-summary>
+      <div class="container">
+        <center>
+          <p id="page-summary">
+            {% block page-summary %}
+              {% blocktrans trimmed %}
+              {% endblocktrans %}
+            {% endblock %}
+          </p>
+        </center>
+      </div>
+    </page-summary>
```

Comment:
I can do that, but fyi I think it's a bad idea to bold the privacy notice. It's plenty visible as is, and bolding just makes it inconsistent with the rest of the pages.